content/en/docs/Tasks/performance-profiling.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Performance Profiling Metrics"
```

Comment:
I can imagine readers finding this page while trying to understand the performance of their own operator versus the OLM operators. Making the title more specific might save them some confusion or at least a click.